shared/provision/username-or-email/index.js

```diff
@@ -0,0 +1,114 @@
+// @flow
+// TODO remove Container
+import * as React from 'react'
+import * as Kb from '../../common-adapters'
+import * as Styles from '../../styles'
+import Container from '../../login/forms/container'
+import * as Constants from '../../constants/provision'
+import {RPCError} from '../../util/errors'
+import {constantsStatusCode} from '../../constants/types/rpc-gen'
+
+type Props = {
+  error: string,
+  inlineError: ?RPCError,
```

Comment:
lets not expose a raw error here. the container can expose inlineErrorText or badUsernameError boolean or something